ssts/segmentation/example.py

```diff
@@ -5,6 +5,8 @@
 import ssts.segmentation.segmentation_watershed as seg_water
 from methods import main_methods as mm  # NOTE SSTS directory must be in PYTHONPATH
 
+Z_SCORE_THRESH = 2.5
```

Comment:
@wesleyktatum This is how you change the value of the z score threshold used for outlier extraction.